### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by stweil**
Signed-off-by: Stefan Weil <sw@weilnetz.de>

### CODE REVIEW

File: 1.1/spec.md [hunk 1]
```diff
@@ -425,7 +425,7 @@ The following properties should be present:
   * syntactically, must be a UNIX-like pathname or http URL (no Windows pathnames)
   * may be relative
   * cannot be resolved to the actual file in general (e.g., if the hOCR file
-    becomes separated from the image fiels)
+    becomes separated from the image files)
```
Comment (amitdo) [inline 1.1/spec.md]: According to the context, I think it should be `image file` and not `image files` .

Comment (amitdo) [reply 1.1/spec.md]: It turns out I fixed this typo more than 5 years ago!
 
https://github.com/kba/hocr-spec/commit/5552893beeebbd218a81a49de72a729fe64626f2

but only in 1.2.

Comment (stweil) [reply 1.1/spec.md]: Thank you for the review and your hint. I fixed that now.

**Comment by amitdo** [comment]
I squashed and merged your commits. Maybe I should have done a simple merge...

**Pull request merged**